src/lib/genesis_constants/genesis_constants.ml

```diff
@@ -45,13 +45,13 @@ module Constraint_constants = struct
   [%%versioned
   module Stable = struct
     module V1 = struct
-      type t = {c: int}
+      type t = {c: int; pending_coinbase_depth: int}
```

Comment:
For future PR involving scan state constants: `pending_coinbase_depth` is calculated using the scan state constants `transaction_capacity_log_2` and `work_delay`. So the depth value must be set using these and not some independent value

Reply:
Thanks, I'll move this logic into `Genesis_constants` as I move those.